src/components/styles/collapsed.module.css

```diff
@@ -0,0 +1,25 @@
+.container {
```

Comment:
lets use BEM with postcss-nesting

Reply:
So, it's a question. Will you use BEM or CSS Modules? As I know, these are different approaches.

Reply:
Maybe will change every class to CSS Modules? Because it's simple and vite uses this with default configuration

Reply:
ok